README.md

```diff
@@ -18,18 +18,32 @@ For running the services that are event sourced, it needs a database for persist
 
 ## Locally running on microk8s:
 
-1. Install microk8s if not already installed: see https://microk8s.io/docs/install-alternatives for instructions
-2. Install scylla-operator on microk8s: see https://operator.docs.scylladb.com/stable/generic.html for instructions
-3. Run command `microk8s kubectl apply -f microApply.yaml`
-4. Check status with `microk8s kubectl get pods -o wide`
-5. Inspect pod using `microk8s kubectl describe pod [pod-name]`
-6. Inspect services using `microk8s kubectl logs [pod-name] -c [service-name]`
+### IMPORTANT NOTE
+For now, local scylla-db services can only be connected to in microk8s by changing a service's `application.conf` to use the *internal* `ClusterIP` of the `scylla-db` service
+For this, `scylla applyForInternalIP.yaml`, must be used in place of `scyllaApply.yaml` in above steps.
+
+## Installation Instructions
+1. Install microk8s if not already installed (: see https://microk8s.io/docs/install-alternatives for instructions)
+   - If on mac, just run `brew install microk8s`
+2. Enable dns with `microk8s enable dns`
+3. Install scylla
+   - Run command `microk8s kubectl apply -f [scyllaAppl/applyForInternalIP]y.yaml`
+4. Run command `microk8s kubectl apply -f microApply.yaml`
+5. Check status with `microk8s kubectl get pods -o wide`
+
+## Exposing Services Instructions
+1. Inspect pod using `microk8s kubectl describe pod [pod-name]`
```

Comment:
where do [pod-name] and [service-name] come from?

Reply:
In `get pods` or `get service` you will see a list of NAME (that is how the column in formatted). I have modified the README accordingly. Service-name it turns out is unnecessary.